pkg/engine/engine.go

```diff
@@ -886,6 +886,32 @@ func getClusterAutoscalerAddonFuncMap(addon api.KubernetesAddon, cs *api.Contain
 	}
 }
 
+func getArcAddonFuncMap(addon api.KubernetesAddon) template.FuncMap {
```

Comment:
I don't think we need this template funcmap, we can use `getAddonFuncMap`, which already has a way to get the config key vals (`ContainerConfig`)

Reply:
Added `ContainerConfigBase64` func